### PR TITLE
Create Exercise 2.5

### PR DESCRIPTION
**Pull request by Diontre**
Here are my codes for the exercise. What are your thoughts?

### CODE REVIEW

File: Exercise 2.5 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#Diontre Davis
+#Public Policy 890
+
+#1. Why does this code not work?
+#my_variable <- 10
+#my_varıable
+#> Error in eval(expr, envir, enclos): object 'my_varıable' not found
+#Answer: The error appeared because the instructions for this assignment were
```
Comment (epticrinduke) [inline Exercise 2.5:8]: This isn't how I answered this. The character "ı" is different from the character "i" used in the word "variable," causing an error.

File: Exercise 2.5 [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#1. Why does this code not work?
+#my_variable <- 10
+#my_varıable
+#> Error in eval(expr, envir, enclos): object 'my_varıable' not found
+#Answer: The error appeared because the instructions for this assignment were 
+# specific enough for R to process.
+
+#2. Tweak each of the following R commands so that they run correctly:
+library(tidyverse)
```
Comment (epticrinduke) [inline Exercise 2.5:12]: Yes, this is what I got!

File: Exercise 2.5 [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ggplot(data = mpg) + 
+  geom_point(mapping = aes (x = displ, y = hwy)) +
+  geom_smooth(aes (x = displ, y = hwy), method = "lm")
+
+#3. If I press Alt + Shift + K/Option + Shift + K, then I am able to see all of
```
Comment (epticrinduke) [inline Exercise 2.5:17]: Same answer for me!

File: Exercise 2.5 [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+#the shortcuts to certain commands on R that can be done using the keyboard. 
+# This can also be found under the "Tool" window.
+
+#4.Answer: The plot that is saved as a png is the bar plot was saves. In the
```
Comment (epticrinduke) [inline Exercise 2.5:21]: Same answer for me!

**Review by epticrinduke — COMMENTED** [review]
With #2-#4, we got the same answers. Any wording difference you want to merge with the 2.5 file? Just noting that merging changes as is would create two different 2.5 files. Did my explanation of #1 make sense?

**Comment by Diontre** [comment]
> With #2-#4, we got the same answers. Any wording difference you want to merge with the 2.5 file? Just noting that merging changes as is would create two different 2.5 files. Did my explanation of #1 make sense?

**Comment by Diontre** [comment]
I understand your explanation for answer #1. Took another look and thought about it to understand

**Pull request closed without merging**